dxp/wysiwyg.md

```diff
@@ -0,0 +1,33 @@
+# WYSIWYG Editors
+
+We have recently worked on reducing variability in the WYSIWYG writing experience. After analyzing current usage of the editors the decision was made to provide a more cohesive and comprehensive experience for each specific use case. Documenting the **WYSIWYG** editors that are available in [Liferay DXP](https://github.com/liferay/liferay-portal) can be useful.
+
+At the time of writing this document, there are three different **WYSIWYG** editors in [Liferay DXP](https://github.com/liferay/liferay-portal):
+
+-   [AlloyEditor](https://alloyeditor.com)
+-   [CKEditor 4](https://ckeditor.com/ckeditor-4/)
+-   [TinyMCE](https://www.tiny.cloud/) - disabled by default
+
+Editors are usually instantiated in jsp editor taglib and in js files.
+
+For most applications the editor that will be used is configured via configuration settings or [`portal.properties`](https://github.com/liferay/liferay-portal/blob/7a8b847a3f3e8bc649d94cb80248623ea2bde5a2/portal-impl/src/portal.properties).
+The default WYSIWYG editor is CKEditor.
+
+If you need to use a WYSIWYG in your application, please identify the different use cases before taking a decision.
+For example, don't use a WYSIWYG editor when you can use an input element.
```

Comment:
I would say it is a bit more nuanced than that. You "can" use an `input` element almost anywhere, but that doesn't mean you should. Instead of:

> For example, don't use a WYSIWYG editor when you can use an input element.

I'd suggest:

> For example, don't use a WYSIWYG editor when you an `input` element would be more appropriate.

Reply:
ok, thanks!